tikv/raftstore/coprocessor.go

```diff
@@ -97,8 +103,11 @@ func (checker *sizeSplitChecker) onKv(obCtx *observerContext, spCheckKeyEntry sp
 func (checker *sizeSplitChecker) getSplitKeys() [][]byte {
 	// Make sure not to split when less than maxSize for last part
 	if checker.currentSize+checker.splitSize < checker.maxSize {
-		if len(checker.splitKeys) != 0 {
-			checker.splitKeys = checker.splitKeys[:len(checker.splitKeys)-1]
+		splitKeyLen := len(checker.splitKeys)
+		if splitKeyLen != 0 {
+			// Help gc
```

Comment:
No need to do this because the checker is a short lived object.

Reply:
ok, thanks.